resources/com.sap.piper/pipeline/cloudSdkContainersMap.yml

```diff
@@ -52,9 +52,10 @@ containerMaps:
     - mavenExecute
   artifactDeployment:
     - nexusUpload
-  whitesourceScan:
-    - mavenExecute
-    - executeNpm
+  security:
+    - fortifyExecuteScan
+    - detectExecuteScan
+    - whitesourceExecuteScan
```

Comment:
What about checkmarx?

Reply:
Ah, I think checkmarx does not use a docker container. sry.

Reply:
No problem, I was also confused that it wasn't present in the containersMap.